binding-to-localhost-recommendation.md

```diff
@@ -0,0 +1,30 @@
+## Binding to Network Interfaces
+### Recommendation: 
+Always bind to localhost rather than to 0.0.0.0 or any interface, unless there is a specific need to do otherwise. Binding to localhost reduces the attack surface of your application by making it only accessible to devices on the same machine.
+Binding to 0.0.0.0 or "all" interfaces can make your application respond on all current and future network interfaces.
+
+**Rationale:**
```

Comment:
I believe that having this section and the next ones is unnecessary. The first two parts of this are already enough, IMO. And this recommendation should be added to the docs/recommendations.md instead of its own file.

Reply:
Okay, I will make the changes right away